### ISSUE TITLE
Eslint gives errors/nothing when run with npm and Rake

### ISSUE THREAD

**Issue by reconstructions** [shakacode/react-webpack-rails-tutorial#364]
When the 'lint' script in package.json is run with `npm run lint`, it gives 82 linting errors.

Running the `rake lint:eslint` task gives something like this:

```
rake lint:eslint
Running eslint via `cd client && $(npm bin)/eslint . -- --ext .jsx,.js`
cd client && $(npm bin)/eslint . -- --ext .jsx,.js
```

But that's it.  

Assuming that the Rake task isn't working, are these real linting errors given your "eslint-config-shakacode" package?  If so, I will fix them, let me know if it would make sense to try and fix them here.

**Comment by justin808** [comment]
@reconstructions It would be great to get the linter updated (the npm package, a PR would be great ), as well as to update this repo. 

I'm looking for somebody to help me with the update to Webpack 2! https://github.com/shakacode/react-webpack-rails-tutorial/issues/336

**Comment by reconstructions** [comment]
Sounds great, I have a commit which gets Eslint to green with minimal changes to the app, I will try to make sure that is ready for a PR.  I also have a couple of months worth of small enhancements which might bolster this repo, for example setting all the ports in an .env file, 12 factor app #3 style.  Plus some extra test coverage which has made it easier for me to refactor the app.  I will start taking a look at Webpack 2 and see if I am fit to contribute there...

**Comment by justin808** [comment]
@reconstructions Any updat?

**Comment by reconstructions** [comment]
Thanks @justin808, I submitted PR #368 which gets Eslint to green a couple days ago, @kaizencodes thought his #336 might also fix this, I was going to pull #336 and see how that works in a couple of my apps, I will let you know.
